kubernetes/zenko/charts/cloudserver/dashboards/cloudserver.json

```diff
@@ -916,7 +906,7 @@
         }
       ],
       "thresholds": "60,75",
-      "title": "RClone OOB Throughput - Average",
+      "title": "Average RClone Operations",
```

Comment:
sorry to be so nitpicky: `Operations` -> `Operations/second(min, hr...)`

Reply:
No not nitpick at all. This is exactly the feedback I was looking for. I've been staring at this so long I can't see the obvious issues anymore.